doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -15,17 +15,17 @@ In summary, **Conformance** testing will be performed as part of cloud infrastru
 - **Verification** is performed using reviews (e.g., manifests) to ensure that the cloud infrastructure or workload is delivered as per implementation specifications.
 - **Validation** confirms the infrastructure or workload meet the expected or desired behaviour by means of automated testing.
 
-**All Terms utilized throughout this chapter are intended to align with LFN Complinace and Verification Committee (CVC) definitions, and their use through CVC documentation, guidelines, and standards.**
+**All Terms utilized throughout this chapter are intended to align with LFN Compliance and Verification Committee (CVC) definitions, and their use through CVC documentation, guidelines, and standards.**
 
 ## Introduction
 
 **Document Purpose**
 
-This chapter includes tracability from requirements to test cases and a test case execution framework to ensure Kubernetes infrastructure meets the design, feature, and capability expectations of the Reference Model (RM) and RA2. Ultimately, RC2 will reduce the amount of time and cost it takes each operator to on-board and maintain cloud infrastructure and CNFs.
+This chapter includes traceability from requirements to test cases and a test case execution framework to ensure Kubernetes infrastructure meets the design, feature, and capability expectations of the Reference Model (RM) and RA2. Ultimately, RC2 will reduce the amount of time and cost it takes each operator to on-board and maintain cloud infrastructure and CNFs.
```

Comment:
@gkunz nitpick: since spelling out RM, we should also spell out RA2 or (preferred) just remove "Reference Model" and the parentheses around RM.

Reply:
done.